frontend/src/assets/locales/ro/translation.json

```diff
@@ -32,7 +34,16 @@
     "members": "Membri",
     "branches": "Filiale si sucursale",
     "departments": "Departamente",
-    "roles": "Roluri"
+    "roles": "Roluri",
+    "error": {
+      "delete": "Eroare la stergerea diviziei"
+    },
+    "delete": {
```

Comment:
we should have modal "modal": {"delete" : {
    "title": "Sterge divizie",
      "description": "Esti sigur ca doresti stergerea diviziei?",
      "success": "Divizie stearsa cu succes",
}}